code/Generators/DocBlockGenerator.php

```diff
@@ -9,7 +9,7 @@
  *
  * @package IDEAnnotator/Generators
  */
-class DocBlockGenerator
+class DocBlockGenerator extends Object
```

Comment:
I imagine this change was made to make the class extensible. I think that's fine, especially given that this module is only used as a dev dependency - not going to slow down production environments.